src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Data.SqlClient.csproj">
+      <SupportedFramework>net46;dnxcore50</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\facade\System.Data.SqlClient.csproj">
+      <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
+    </ProjectReference>
+
+    <ProjectReference Include="win\System.Data.SqlClient.pkgproj">
+        <PackageAlias>win.System.Data.SqlClient</PackageAlias>
+    </ProjectReference>
+    <ProjectReference Include="unix\System.Data.SqlClient.pkgproj" />
+
+    <!-- Ideally we'd place these in the windows project, but nuget cannot handle
+         recursive runtime runtime dependencies -->
```

Comment:
This is actually no longer any different than the rest of our shim dependencies (eg: System.IO.Filesystem unix implementation depends on the system.native shim and we put that in the runtime.unix.S.IO.FS package).  We've taken a hard-requirement on the uber-runtime.json being present.  As such we could move these back down to the windows package where they belong.

Not in this PR though.